frontend/views/components/AvatarUpload.vue

```diff
@@ -57,7 +57,7 @@ export default ({
       try {
         const { selector, contractID, key } = this.sbpParams
         await sbp(selector, { contractID, data: { [key]: picture } })
-        this.$refs.picture.setFromBlob(fileReceived)
+        // this.$refs.picture.setFromBlob(fileReceived)
```

Comment:
Why is this being commented out?

I think that could cause problems... and it seems unrelated to the issues in the PR description.

Reply:
It's one of the solution of the issue #1425.
As I described in the issue, the problem is that the group avatar is not updated when user switch group after upload picture once.
If user upload picture, the `Avatar` component loses the connect to the group profile picture because we set it from the Blob not from the group picture link.
Here, if user switch group, the group picture is not updated because it's already lose the connection

Reply:
Ah, got it! Can you please add the following comment above this line then:

```js
// calling `setFromBlob` seems unnecessary here since the bound :avatar
// parameter should get updated after the upload completes.
// also, calling it here prevents the avatar from switching in Group Settings if we
// just uploaded a new image and then switch groups (see #1425)
```